extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/src/main/java/org/eclipse/smarthome/binding/bosesoundtouch/BoseSoundTouchBindingConstants.java

```diff
@@ -71,11 +72,29 @@
     public static final String CHANNEL_NOWPLAYING_STATIONLOCATION = "nowPlayingStationLocation";
     public static final String CHANNEL_NOWPLAYING_STATIONNAME = "nowPlayingStationName";
     public static final String CHANNEL_NOWPLAYING_TRACK = "nowPlayingTrack";
-
-    // Device configuration parameters;
-    public static final String DEVICE_PARAMETER_HOST = "host";
+    public static final String CHANNEL_NOTIFICATION_SOUND = "notificationsound";
 
     // Device information parameters;
     public static final String DEVICE_INFO_NAME = "INFO_NAME";
     public static final String DEVICE_INFO_TYPE = "INFO_TYPE";
+    
+    public static class BoseSoundTouchConfiguration {

```

Comment:
We usually have those a standalone classes, not being embedded in the constants.

Reply:
OK - but may I put it in an exported package?

Reply:
Or move everything in internal packages (what we now do in the skeleton as well).

Reply:
But I will need to access the config param names outside the binding (and anyway they are also visible in the config.xml) - I think it would be better to put the constants in an exported package

Reply:
Sure, feel free.

Reply:
you mean - extract the constants from the ```BoseSoundTouchConfiguration``` and move them to ```BoseSoundTouchBindingConstants``` or externalize the ```BoseSoundTouchConfiguration``` class and put in an exported package ?

Reply:
Jus externalize it in the same package (which already is exported).